src/test/ui/suggestions/recover-invalid-float-invalid.rs

```diff
@@ -0,0 +1,21 @@
+fn main() {}
```

Comment:
A brief comment at the top of the file describing what's being tested would be good.